core/src/test/java/org/testcontainers/dockerclient/EventStreamTest.java

```diff
@@ -0,0 +1,61 @@
+package org.testcontainers.dockerclient;
+
+import com.github.dockerjava.api.DockerClient;
+import com.github.dockerjava.api.model.Event;
+import com.github.dockerjava.core.command.EventsResultCallback;
+import org.junit.Test;
+import org.rnorth.visibleassertions.VisibleAssertions;
+import org.testcontainers.DockerClientFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.startupcheck.OneShotStartupCheckStrategy;
+
+import java.io.IOException;
+import java.time.Instant;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+/**
+ * Test that event streaming from the {@link DockerClient} works correctly
+ */
+public class EventStreamTest {
+
+    @Test
+    public void test() throws IOException, InterruptedException {
+        try (final DockerClient client = DockerClientFactory.instance().client()) {
+
+            Instant startTime = Instant.now();
+
+            final AtomicBoolean received = new AtomicBoolean(false);
+
+            EventsResultCallback callback = new EventsResultCallback() {
+                @Override
+                public void onNext(Event event) {
+                    if (event.getFrom().equals("alpine:latest") && event.getStatus().equals("create")) {
+                        received.set(true);
+                    }
+                }
+            };
+
+            GenericContainer container = new GenericContainer<>("alpine:latest")
```

Comment:
After https://github.com/testcontainers/testcontainers-java/pull/1886/files#r325612659, you can remove the custom image here and use the "default" one (alpine 3.6 AFAIR)